.coveragerc

```diff
@@ -0,0 +1,8 @@
+[report]
```

Comment:
This file seems already present in #1.
Same remark for several files of this PR.

Anyway, #2 should be rebased after merge of #1.

Reply:
Hello, 
Yes, #1 was set as a blocker for this PR , that is #1 is a base branch for this PR. I will remerge #1 into #2 after #2 is merge to solve conflicts.

Reply:
My advice is to **remove all duplicated files** from PR 2:
- to lighten the diff, avoiding to review many unmodified files;
- to avoid future conflicts when PR 1 will be merged.

In fact, PR 2 needs to add only 4 new files:
```
pyriemann_qiskit/classification.py
tests/test_classification.py
examples/ERP/README.txt
examples/ERP/plot_classify_EEG_quantum.py
```
All other files already exist in PR 1.

Reply:
Hum... there will be likely no merge conflict because this branch is issued from the one of PR 1 and they share the same commit history. Also I need some of the files defined in the underlying architecture, such as __init__.py or setup.py. 
Would it be more comfortable to pursue the review in this draft: https://github.com/gcattan/pyRiemann-qiskit/pull/1
(branch is the same so any modification there will be applicable here)?

Reply:
All these PR are a little bit too messy for me. 
But I prefer continue on this PR, even though the diff is not easy to review.

Reply:
I understand and will take into account your consideration for future PRs. 
Once again, thank you very much for your hard work in reviewing this code. I know this is already a lot of work and I highly appreciate your time and help to improve the quality of this implementation.